src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -1865,19 +1865,7 @@ private static void GetAccessibilityAndModifiers(SyntaxTokenList modifierList, o
                         break;
 
                     case SyntaxKind.ProtectedKeyword:
-                        if (accessibility == Accessibility.Private)
-                        {
-                            accessibility = Accessibility.ProtectedAndInternal;
-                        }
-                        else if (accessibility == Accessibility.Internal)
-                        {
-                            accessibility = Accessibility.ProtectedOrInternal;
-                        }
-
```

Comment:
❗️ This should say `else`, right?

Reply:
i could make that change as well :)  but this was just a behavior-preserving change :)

Reply:
#resolved